src/app.rs

```diff
@@ -81,6 +82,7 @@ impl Application {
                     format!("file_source={}", level),
                     "tower_limit=trace".to_owned(),
                     format!("rdkafka={}", level),
+                    format!("buffers={}", level),
```

Comment:
A note: I checked the `buffers` crate and there are no logs at the info level, only debug/trace and the new addition of a warning message in this PR.. so the effective default behavior will be the users only see the new message we've added here if they sleuth out the new `disk_v2` buffer type and configure it.